vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -473,6 +473,13 @@ function Update-Registry {
             Write-Log "The current value of 3767762061 is $currentValue"
         }
         Set-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Policies\Microsoft\FeatureManagement\Overrides" -Name 3767762061 -Value 1 -Type DWORD
+
+        Write-Log "Enable 1 fix in 2024-01B"
+        $currentValue=(Get-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Policies\Microsoft\FeatureManagement\Overrides" -Name 1102009996 -ErrorAction Ignore)
+        if (![string]::IsNullOrEmpty($currentValue)) {
+            Write-Log "The current value of 1102009996 is $currentValue"
+        }
+        Set-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Policies\Microsoft\FeatureManagement\Overrides" -Name 1102009996 -Value 1 -Type DWORD
```

Comment:
Add test in vhdbuilder/packer/test/windows-vhd-content-test.ps1

Reply:
Good catch. Missed it in rebasing.